frontend/src/routes/post/[id]/+page.svelte

```diff
@@ -19,17 +20,15 @@
         } catch (err: unknown) {
             if (err instanceof ApiError) {
                 if (err.status === 403) {
-                    const t: ToastSettings = {
-                        message: "You can't delete this post",
+                    getToastStore().trigger({
+                        message: "You aren't logged in",
                         background: 'variant-filled-error'
-                    };
-                    getToastStore().trigger(t);
+                    });
                 } else if (err.status === 401) {
-                    const t: ToastSettings = {
+                    getToastStore().trigger({
                         message: "You aren't logged in",
                         background: 'variant-filled-error'
-                    };
-                    getToastStore().trigger(t);
+                    });
```

Comment:
Put `getToastStore()` in a const variable somewhere top level instead of having to write this out all the time. Also, preferably make it a util so you also don't have to rewrite `background: 'variant-filled-error' constantly too.

Reply:
Ah yea, thanks for noticing that!

Reply:
Probably going to skip out on making a util for it as it feels more "explicit" for others reading to have it there.